build.rs

```diff
@@ -1,3 +1,23 @@
-fn main() {
-    git_version::set_env_with_name("SYMBOLICATOR_GIT_VERSION");
+use std::io;
+use std::process::{Command, Stdio};
+
+fn main() -> Result<(), io::Error> {
```

Comment:
Actually let's catch down all errors here. @tonyo explicitly sets `SYMBOLICATOR_GIT_VERSION` in the dockerfile because there is either no git folder or no git installed.